nest/03.advanced_concepts.md

```diff
@@ -0,0 +1,70 @@
+# 3 Advanced concepts
+In this chapter, we will detail some advanced concepts related to `nest`, which will help understanding how `nest` works, but aren't strictly needed for a daily use of `nest`.
+On the other hand, readers interested in `finest` should read this chapter, because those concepts are important to understand how `finest` works.
+
+## 3.1 Managing repositories
+### 3.1.1 The official repositories
+The official repositories are the ones maintained by the Raven-OS team.
+For now, only one is available: `stable`, which is the one enabled by default when installing Nest.
+It contains all the stable and safe-to-use packages for Nest, and is maintained by the Raven-OS team.
+
+### 3.1.2 The unofficial repositories
+There are also unofficial repositories for Nest.
+They are not being maintained by the Raven-OS team or someone trusted by the team.
+You must decide whether to trust their maintainers and take full responsibility for any consequences of using any unofficial repository.
+
+### 3.1.3 Configuring repositories
+The repositories configuration is in the `config.toml` file of Nest.
+It would look something like this:
+
+```toml
+# Stable repository
+[repositories.stable]
+mirrors = ["https://stable.raven-os.org"]
+
+# Beta repository (uncomment to enable)
+# [repositories.beta]
+# mirrors = ["https://beta.raven-os.org"]
+
+# Nightly repository (uncomment to enable)
+# [repositories.nightly]
+# mirrors = ["https://nightly.raven-os.org"]
+```
+
+A mirror is basically a copy of all datas from the main server.
+They exist for redundancy and speed.
+Mirrors are spread all over the globe in strategic locations.
+If you live in Sweden and use a mirror from Sweden, the download speed should be faster than if you used a mirror from Australia for example.
+The point of mirrors is to choose the nearest one from your location, so you'll download packages as fast as possible.
+It's advised to have several mirrors for each repository Nest uses, because a mirror can be down or broken for a small period of time, or just be down forever.
```

Comment:
I don't have a better formulation in my mind right now, but this does sound clunky to me.

Reply:
```suggestion
Because a mirror can be down or broken for a short period of time, or just forever, t's advised to have more than one mirror for each repository used by Nest.
```

Reply:
(edited your comment to fix typo)